specs/signing-scheme.md

```diff
@@ -48,35 +50,35 @@ This attribute dictates the rest of signature schema - the set of signed and uns
 
 Both `notary.x509` and `notary.x509.signingAuthority` signing schemes use the similar signature schema (set of signed and unsigned attributes) with the following differences.
 
-* `notary.x509` MUST use a countersignature from trusted source to determine authentic signing time (timestamp).
-This is supported through the Timestamp signature unsigned attribute in the signature envelope. Currently Notary Project uses a [RFC3161](ietf-rfc3161) compliant TSA signature for this purpose.
+* `notary.x509` can only use a timestamp countersignature to prove that the signature was generated before the time point been stamped by the `Timestamp Authority (TSA)`.
+This is supported through the OPTIONAL *Timestamp Signature* unsigned attribute in the signature envelope. Currently the Notary Project only supports a [RFC3161][ietf-rfc3161] compliant Timestamp for this purpose. When not provided, every certificate in the certificate chain MUST be within its validity period at the time of verification.
 
-* `notary.x509.signingAuthority` MUST use a timestamp attribute that is generated by a signing service as part of the original signature itself to determine authenticated signing time (timestamp).
-This is supported through the *Authentic Signing time* attribute in the signature envelope.
+* `notary.x509.signingAuthority` MUST use a date/time attribute that is generated by the signing service as part of the original signature to determine `authentic signing time`.
```

Comment:
The term _timestamp_ and _date/time_ are different where _timestamp_ has time zone info but datetime does not.

Reply:
They are two completely different terms. A `timestamp` is a stamp of a time point, it's not a time point itself. Wherease, `date/time` means a time point. For `notary.x509.signingAuthority`, because we want to determine the `authentic signing time` here, what we need is a time point instead of a `timestamp`.